python/py-typing_extensions/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup python 1.0
 name                py-typing_extensions
 version             3.10.0.2
 license             PSF
-platforms           darwin
```

Comment:
@jmroot just out of curiosity, I thought we always added `platforms`; also the guide suggest it's required ("A list of the platforms on which the port is expected to work. Required, but not interpreted in any way by MacPorts at this time; it is purely informational for users."

has something changed here?

Reply:
2.7 added a default value of `darwin` for platforms. So I guess the docs should be updated to indicate that it only needs to be set to non-default values.